invenio_rdm_records/resources/serializers/dublincore/__init__.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Dublin Core Serializers for Invenio RDM Records."""
+
+from dcxml import simpledc
+from flask_resources.serializers import MarshmallowJSONSerializer, \
+    SerializerMixin
+
+from .schema import DublinCore
+
+
+class DublinCoreJSONSerializer(MarshmallowJSONSerializer):
+    """Marshmallow based Dublin Core serializer for records."""
+
+    def __init__(self, **options):
+        """Constructor."""
+        super().__init__(schema_cls=DublinCore, **options)
+
+    def dump_one(self, obj):
+        """Dump the object with extra information."""
+        json = self._schema_cls().dump(obj)
+        return json
+
+
+class DublinCoreXMLSerializer(SerializerMixin):
+    """Marshmallow based Dublin Core serializer for records.
+
+    Note: This serializer is not suitable for serializing large number of
+    records.
+    """
+
+    def __init__(self, **options):
+        """Constructor."""
+        self._schema_cls = DublinCore
+
+    def serialize_object(self, obj) -> str:
```

Comment:
It is nice to use the type hints, but we are not really using it accross Invenio... so just to make it consistent :)
```suggestion
    def serialize_object(self, obj):
```

Reply:
Let's use type hints!!!! Revolution! :D

Reply:
I'm up for it!